components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java

```diff
@@ -100,7 +102,7 @@
     private String uuid;
     private String policyPartials;
     private String policyGroups; //Policy Groups Id's list
-    private String javaPolicies; //Java policies(handlers) List
+    private JSONArray javaPolicies; //Java policies(handlers) List
```

Comment:
In the latest code type has been changed to "String" to except comma separated ID list